src/index.html

```diff
@@ -9,10 +9,46 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/index.css"
```

Comment:
The href attribute should link to a `.scss` file (index.scss) according to the project structure, not a `.css` file. Please correct the file extension to ensure proper preprocessing of SCSS into CSS.